andela/elven-ah-client
# #160056177 set up sass and bootstrap and update readme

#### What does this PR do?
Set up sass processing and bootstrap on the client and update readme
#### Description of Task to be completed?
- Set up sass processing on the client
- Set up bootstrap 4.0 framework on the client
- Update readme.md
#### How should this be manually tested?
- create a .scss file and write some styling, 
- Try using some bootstrap properties in .js file
- import the created .scss file in a .js file
- run npm start
- You should see your style being effected without any compilation error
#### What are the relevant pivotal tracker stories?
[#160056177](https://www.pivotaltracker.com/story/show/160056177)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/33626790/44914033-e2c93600-ad26-11e8-8d93-4cd6474a93e9.png)
